eos-live-boot-overlayfs-setup

```diff
@@ -17,6 +17,17 @@ find_storage_partition() {
 	local root_partition=$(blkid --output device --match-token "${bootdev}")
 	[ -b "${root_partition}" ] || return 1
 
+
+	dumpexfat -f /endless/persistent.img ${root_partition} > /dev/null 2>&1
+	if [[ $? = 0 ]]; then
+		udevadm settle
```

Comment:
Is this needed because `dumpexfat` accesses the partition device directly?

Reply:
No, it's because eos-map-image-file uses lsblk